src/pages/dev/amplifier/chain-integration/integrate-a-chain.mdx

```diff
@@ -68,35 +71,37 @@ Install the `v0.35.5 axelard` binary. You can either download the pre-built vers
     !faucet devnet-amplifier [your axelar wallet address]
     ```
 
-
 1. Query your account balance by running:
 
     ```bash
     axelard q bank balances [your axelar wallet address] --node http://devnet-amplifier.axelar.dev:26657
     ```
+</tab-item>
 
-## Onboard a chain to Amplifier
+</tabs>
 
-Once you are running `ampd` and `tofnd`, you can onboard a chain to the Amplifier by instantiating the `gateway`, `voting verifier`, and `multisig prover` contracts.
+## Onboard a chain with Amplifier
 
-You can either use the existing deployed contracts and on the devnet via their `code_id`s, or build and store these contracts from the `axelar-examples` repo. A `code_id` is a CosmWasm identifier of a stored contract that can be instantiated. **You will need to instantiate the contracts no matter which option you choose.**
+Once you are running `ampd` and `tofnd`, you can onboard a chain with the Amplifier by instantiating the `gateway`, `voting verifier`, and `multisig prover` contracts.
```

Comment:
This step should be last, after the contracts are instantiated as `ampd` and `tofnd` are only for verifying for an integration.